src/include/tpetra_wrapper_types.h

```diff
@@ -136,31 +140,42 @@ class TpetraPartitionMap {
     KOKKOS_INLINE_FUNCTION
     size_t extent() const;
     
+    KOKKOS_INLINE_FUNCTION
     int getLocalIndex(int global_index) const;
+    
+    KOKKOS_INLINE_FUNCTION
+    long long int getGlobalIndex(int local_index) const;
 
-    int getGlobalIndex(int local_index) const;
+    KOKKOS_INLINE_FUNCTION
+    long long int getMinGlobalIndex() const;
 
+    KOKKOS_INLINE_FUNCTION
+    long long int getMaxGlobalIndex() const;
+    
+    KOKKOS_INLINE_FUNCTION
     bool isProcessGlobalIndex(int global_index) const;
-
+    
+    KOKKOS_INLINE_FUNCTION
     bool isProcessLocalIndex(int local_index) const;
 
     // Method returns the raw device pointer of the Kokkos DualView
     KOKKOS_INLINE_FUNCTION
     T* device_pointer() const;
 
     // Method returns the raw host pointer of the Kokkos DualView
-    KOKKOS_INLINE_FUNCTION
     T* host_pointer() const;
 
-    // Method returns kokkos dual view
-    KOKKOS_INLINE_FUNCTION
-    TArray1D get_kokkos_dual_view() const;
+    void print() const;
 
-    // Method that update host view
-    void update_host();
+    // // Method returns kokkos dual view
```

Comment:
Can these be removed? If so, please do.

Reply:
Yea I cleaned up a couple things there I'll push once I can get a new CRS matrix wrapper to at least clean compile